docs: update main readme (#1227)
rx-angular/rx-angular: README.md

This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.

## Benefits

>  - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
>  - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
>  - ✔ Lean and simple: No boilerplate guaranteed
>  - ✔ Well typed and tested
>  - ✔ Backwards compatible: support for Angular > v11
> 

Author: ```suggestion
 - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
 - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
 - ✔ Lean and simple: No boilerplate guaranteed
 - ✔ Well typed and tested
 - ✔ Backwards compatible: support for Angular > v11

> **❗ Warning**
> Expect no migration scripts for any change in `experimental` folders
```

This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.

## Benefits

>  - ✔ Migration scripts form beat versions on! Just run `ng update @rx-angular/???`
>  - ✔ No boilerplate garantieed
>  - ✔ Performance focused and small bundle size through sub packages
>  - ✔ Well typed and tested
>  - ✔ support for Angular > v11
>  - ❗ expect no migration scripts for changes in `experimental` folders

Reviewer: ```suggestion
 - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
 - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
 - ✔ Lean and simple: No boilerplate guaranteed
 - ✔ Well typed and tested
 - ✔ Backwards compatible: support for Angular > v11
```

Author: > - ❗ expect no migration scripts for changes in `experimental` folders

The reason fot 1 exclamation mark was to put the rest in contrast. A small trick from marketing. "Only one thing next to all the good stuff. 🤯"

Author: I like the reordering! Definitely better now.

Reviewer: sorry, but having no migration scripts is no 'benefit' thats why i removed it

This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.

## Benefits

 - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
 - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
 - ✔ Lean and simple: No boilerplate guaranteed
 - ✔ Well typed and tested
 - ✔ Backwards compatible: support for Angular > v11

> **❗ Warning**
> Expect no migration scripts for any change in `experimental` folders
## Used by
<table style="width:100%">
  <tr>
    <td><img src="https://clickup.com/landing/images/logo-clickup_color.svg"></td>
    <td><img src="https://media-exp1.licdn.com/dms/image/C4D0BAQEexCZaCyeDYg/company-logo_200_200/0/1626433899897?e=1652918400&v=beta&t=zjTJehE51V39-IaRRaaK_uBzLqcdTr2wsm7YpLLleUQ"></td>
    <td><img src="https://avatars.githubusercontent.com/u/1733746?s=200&v=4"></td>
  </tr>
   <tr>
    <th>Large scale application</th>
    <th>Medium size project</th>
    <th>Small project</th>
  </tr>
  <tr>
    <td>
      Url: clickup.com <br/>
      LOC: <br/>
      Used in Platforms: Web<br/>
      Total Components: <br/>

>       Total Services: <br/>

Reviewer: should we add all this stuff to the table if there is no data for any of them? ^^

- [📦@rx-angular/cdk](https://github.com/rx-angular/rx-angular/tree/main/libs/cdk/README.md) - Component Development Kit
- [📦@rx-angular/state](https://github.com/rx-angular/rx-angular/tree/main/libs/state/README.md) - Imperative&Reactive Component State-Management

> - [📦@rx-angular/template](https://github.com/rx-angular/rx-angular/tree/main/libs/template/README.md) - High-Performance Non-Blocking Rendering

Reviewer: ```suggestion
- [📦@rx-angular/template](https://github.com/rx-angular/rx-angular/tree/main/libs/template/README.md) - High-Performance Rendering, Concurrent mode
```
